src/mlir/verona-mlir.cc

```diff
@@ -52,16 +53,32 @@ namespace
   // Output file
   cl::opt<std::string> outputFile("o", cl::init(""), cl::desc("Output file"));
 
-  // Grammar file is not optional
-  std::string grammarFile;
+  cl::opt<bool> splitInputFile(
+    "split-input-file",
+    cl::desc("Split the input file into pieces and "
+             "process each chunk independently"),
+    cl::init(false));
 
-  // Set defaults form command line arguments
-  void cmdLineDefaults()
-  {
-    // Default input is stdin
-    if (inputFile.empty())
-      inputFile = "-";
+  cl::opt<bool> verifyDiagnostics(
```

Comment:
there are too many things called `verifyDiagnostics`. This global variable, the driver's method, the driver's flag (which uses underscore to differentiate). This makes for confusing names and get hard to follow.